util/logutil/log.go

```diff
@@ -111,12 +109,33 @@ func InitLogger(cfg *LogConfig) error {
 		return errors.Trace(err)
 	}
 
-	// init logger for grpc debugging
+	err = initGRPCLogger(cfg)
+	if err != nil {
+		return errors.Trace(err)
+	}
+
+	return nil
+}
+
+func initGRPCLogger(cfg *LogConfig) error {
+	// copy Config struct by assignment
+	config := cfg.Config
+	// hack: force stdout
+	config.File.Filename = ""
```

Comment:
Why do we need to force output grpc log to stdout?

Reply:
The old logic directs all grpc log to std. Please take a look at https://github.com/pingcap/tidb/blob/4f31cb49f083870f6b8e0f1ecc9183bc40c4ba42/tidb-server/main.go#L592

Reply:
Nice catch. Then there is a problem: the old logic output to `stderr`. In the tiup deployment, stderr will be piped to the `tidb_stderr.log` file eventually. But if you redirect them to the stdout as in this PR, then you will just lose your outputs, as stdout is not collected anywhere..

Reply:
If you find it hard to print to stderr then I think it is fine to print in the log file. There is no much difference from my perspective.

Reply:
fixed